index.html

```diff
@@ -0,0 +1,233 @@
+                <div class="menu-desktop">
+                    <div class="menu-atlas">
+                        <img src="./img/logo.png" alt="Menu Icon">
+
+                    </div>
+                    <ul class="menu-ul">
+                        <li class="menu-li">
+                            <a href="./about.html">About</a>
+                        </li>
+                        <li class="menu-li">Tickets</li>
+                        <li class="menu-li">Join</li>
+                        <li class="menu-li">Sponsor</li>
+                        <li class="menu-li">News</li>
+                        <li class="menu-li cc">CC Campaign</li>
+
+                    </ul>
+                </div>
+
+            </nav>
+        </div>
+
+    </header>
+    <main>
+        <section>
+            <div class="welcome">
+                <div class="intro">
+                    <h3> "Hello! Sharing World"</h3>
+                    <h1> "CREATIVE COMMONS <br> GLOBAL SUMMIT <br> 2015"</h1>
+                    <p>Praesent porttitor, mi quis pulvinar rutrum, nunc dolor rutrum enim, eu tempor nulla felis eu
+                        orci.
+                        Ut eu turpis nisi. Pellentesque convallis, purus vitae mollis ultrices, ex lectus consectetur
+                        nulla,
+                        et blandit ipsum nibh sit amet ipsum.</p>
+                    <h2> 2015.10.15(THU) ~ 16(FRI) <BR> @ National Museum of Korea, Art Center Nabi and More </h2>
+
+
+
+                </div>
+            </div>
+        </section>
+    </main>
+
+
+    <section id="about">
+        <div class="lets-grow">
+            <h3 class="lets-grow-title">Main Program</h3>
+            <ul class="programs">
+                <li class="Lecture item">
+                    <img src="./img/Tools.png" alt="Tools">
+                    <h4>Lecture</h4>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+
+                </li>
+                <li class="cc-Exhibition item">
+                    <img src="./img/Bulb.png" alt="Bulb">
+                    <h4>CC Exhibtion</h4>
+                    <p>Lorem ipsum dolor sit amet.</p>
+
+                </li>
+                <li class="forum item">
+                    <img src="./img/Handshake.png" alt="Forum">
+                    <h4>Forum</h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+
+                </li>
+                <li class="workshop item">
+                    <img src="./img/Medal.png" alt="Medal">
+                    <h4>Workshop</h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+
+                </li>
+                <li class="ignite item">
+                    <img src="./img/Handshake.png" alt="Handshake">
+                    <h4> CC Ignite</h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, corporis!</p>
+
+                </li>
```

Comment:
- [OPTIONAL] Kindly make sure to align your cards correctly in order to look the same as the provided design
 
     - ### YOUR DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/203947898-9558be07-6c9c-474d-9738-fa589a1b6724.png)

     - ### PROVIDED DESIGN
     
   ![image](https://user-images.githubusercontent.com/87197412/203948123-46cdc3c9-d658-4f68-9b58-519b41c39e3e.png)